packages/@react-stately/menu/src/useMenuTriggerState.ts

```diff
@@ -34,16 +36,28 @@ export function useMenuTriggerState(props: MenuTriggerProps): MenuTriggerState
   let overlayTriggerState = useOverlayTriggerState(props);
   let [focusStrategy, setFocusStrategy] = useState<FocusStrategy>(null);
 
+
+  let [disableClosing, setDisableClosing] = useState<boolean>(false);
```

Comment:
What is the reason for this state? Can we avoid it?

Reply:
hmm i'm not sure what i changed since I first wrote this, but I appear to not need it anymore... it used to have an issue where the menu would close when the subdialog opened